README.md

```diff
@@ -2,11 +2,12 @@
 
 ## Usage
 
-Run `./deploy.sh` to create Hindsight's underlying infrastructure in AWS. Please override
-the `DbPassword` parameter value for your own security's sake.
+Run `./deploy.sh` to create Hindsight's underlying infrastructure in AWS. If not specified
+as parameter overrides to the CloudFormation template, secret values for the Postgres RDS
+instance and the Grafana admin passwords will be randomly generated by `openssl`.
 
```

Comment:
I'm unsure you want to pass in the profile name because the profile itself gets created by the cf. You'll see it names it hindsight-user-role-$ENV. Theres a chicken egg problem here where the user either needs to have that created automatically for them, or we need to just plan around this thing only being used for [dev|prod].  Wondering if @jdenen has any ideas.

Reply:
Wait, now i think i understand what you're doing.  If that works, I think its fine.

Reply:
If you could use a heredoc, i think it would be much easier to understand though.